sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -1488,6 +1544,42 @@ pi_result _pi_queue::executeOpenCommandList(bool IsCopy) {
   return PI_SUCCESS;
 }
 
+// NOTE: this function must be called before createAndRetainPiZeEventList.
```

Comment:
Then it should just be called in the beginning of createAndRetainPiZeEventList (or better inline there)

Reply:
Done.